### ISSUE TITLE
recipes/LibriSpeech/ASR/transformer validation problem

### ISSUE THREAD

**Issue by Yuanyuan-888** [speechbrain/speechbrain#1288]
Hi, I am using 4 V100 32GB GPU to train a Dutch ASR model, I use recipes/LibriSpeech/ASR/transformer/hparams/transformer.yaml recipe. I set batch size 32. But after the 1st epoch, when validation,  the default setting in transformer.yaml : valid_dataloader_opts: batch_size: 1

1 batch size for 4 GPU, that does not work. 

So I change the valid_dataloader_opts: batch_size: 1 -->valid_dataloader_opts: batch_size: 4

But I have 409 sentences in the validation set, so when it comes to the last step, 409%4 = 1, so again 1 batch size for 4 GPU.
![image](https://user-images.githubusercontent.com/67212823/152502974-0b3a1d24-e184-4e72-bed3-7c5e2aa1d7f3.png)

Then when I use ctc_decode 0.40 at the validation stage, what batch size of validation should be?

Thanks in advance!

**Comment by Yuanyuan-888** [comment]
I just set the 
valid_dataloader_opts:
    batch_size: !ref <batch_size>

and my batch_size is 32, It can validate now.
